scripts/deploy/main.bicep

```diff
@@ -271,7 +271,7 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         value: 'Warning'
       }
       {
-        name: 'Logging:LogLevel:SemanticKernel.Service'
+        name: 'Logging:LogLevel:CopilotChat.WebApi'
         value: 'Warning'
       }
```

Comment:
I notice you updated the bicep file but not the associated JSON ARM template.  Please synch both!

Reply:
- [ ] - ![image](https://github.com/microsoft/chat-copilot/assets/128042976/85bb7556-3e48-432b-974c-3cec2350be74)__[![image](https://github.com/microsoft/chat-copilot/assets/128042976/dd28a856-324e-40eb-82d4-fe203032e757)]()